sentry-core/src/main/java/io/sentry/core/SentryOptions.java

```diff
@@ -28,6 +28,7 @@
   private String environment;
   private Proxy proxy;
   private Double sampling;
+  private String inAppPrefix = "io.sentry";
```

Comment:
`inApp` is a list so that the user can add  the package of his app and other libraries that he'd like to show up in Sentry. I believe we shouldn't show Sentry code in the stacktrace for users by default. They can always hit `show all` to see that.